lua/gitblame/init.lua

```diff
@@ -26,7 +30,7 @@ local current_author
 ---@type boolean
 local need_update_after_horizontal_move = false
 
---- This shouldn't be used directly. Use `get_date_format` instead.
+---This shouldn't be used directly. Use `get_date_format` instead.
```

Comment:
The official docs say that no space is needed, and adding a space adds one whitespace character to the annotation

before:

![image](https://github.com/f-person/git-blame.nvim/assets/35381535/06e8770e-c391-4aa0-b9b3-62582ca1ad20)

after:

![image](https://github.com/f-person/git-blame.nvim/assets/35381535/b6db53c6-9ff0-43a9-a146-e549b2158340)